test/integration/verify/controls/azure_db_migration_services.rb

```diff
@@ -0,0 +1,12 @@
+service_name = attribute(:inspec_db_migration_service_name, default: nil)
+sku_name = attribute(:inspec_db_migration_service_sku_name, default: nil)
+location = attribute(:location, default: nil)
+
+control 'azure_db_migration_services' do
+  describe azure_db_migration_services do
```

Comment:
After the control block, we should add the impact and the description. This is how we do in was. Do we have to do the same?

Reply:
it is not mandatory ..